jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt

```diff
@@ -12,36 +12,36 @@ import software.amazon.awssdk.services.lambda.model.Runtime
 import software.aws.toolkits.jetbrains.services.lambda.sam.SamCommon
 import software.aws.toolkits.resources.message
 
-class SamInitRunner(
-    private val name: String,
-    private val outputDir: VirtualFile,
-    private val runtime: Runtime,
-    private val location: String? = null,
-    private val dependencyManager: String? = null
-) {
-    fun execute() = ApplicationManager.getApplication().runWriteAction {
+object SamInitRunner {
```

Comment:
not disagreeing but what was the motivation behind moving this from a class to an object?

Reply:
It annoyed me, that is my only defense

Reply:
I can change revert the file, no biggie

Reply:
no no - I think for something with no state it's probably fine especially since it's just created in place. My hesitation with using objects too much is it promotes the `static` behaviour which then becomes hard to test - in this case I think object is better cos it better illustrates how it's actually used.